lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -571,70 +571,130 @@ describe('BasicLTILaunchApp', () => {
           assert.isFunction(annotationActivityCalls[0].args[1]);
         });
 
-        it('submits a submission when `annotationActivity` event emitted', async () => {
-          renderLTILaunchApp();
-          assert.isFalse(
-            fakeApiCall.calledWith(
-              sinon.match({ path: '/api/lti/submissions' })
-            )
-          );
-          const annotationActivityCalls = getOnActivityCalls(fakeRpcServer.on);
-          const callback = annotationActivityCalls[0].callback;
-          callback('create', {});
-          await new Promise(resolve => setTimeout(resolve, 1));
-
-          assert.calledOnce(fakeApiCall);
-          assert.calledWith(
-            fakeApiCall,
-            sinon.match({ path: '/api/lti/submissions' })
-          );
-
-          const apiCall = fakeApiCall.getCall(0);
-          assert.deepEqual(apiCall.args[0], {
-            authToken: 'dummyAuthToken',
-            path: '/api/lti/submissions',
-            data: {
-              submitted_at: undefined,
-              ...fakeConfig.canvas.speedGrader.submissionParams,
-            },
-          });
-        });
-
-        it('deregisters callback after submission is made', async () => {
-          renderLTILaunchApp();
-
-          const annotationActivityCalls = getOnActivityCalls(fakeRpcServer.on);
-          const callback = annotationActivityCalls[0].args[1];
-          callback('create', {});
-          await new Promise(resolve => setTimeout(resolve, 1));
-
-          assert.calledOnce(fakeApiCall);
-          assert.calledWith(
-            fakeApiCall,
-            sinon.match({ path: '/api/lti/submissions' })
-          );
-          assert.calledOnce(fakeRpcServer.off);
-          assert.calledWith(fakeRpcServer.off, 'annotationActivity', callback);
-        });
-
-        it('submits a submission with annotation-activity date if provided', async () => {
-          renderLTILaunchApp();
+        context(
+          '`annotationActivity` event that qualifies for submission',
```

Comment:
I added a wrapping `context` block here and updated to use `delay` utility but tests already existed here.